### PR TITLE
OCM-199 | feat: Add shared pvc fields to `cluster describe` cmd

### PR DESCRIPTION
**Pull request by marcolan018**
https://issues.redhat.com/browse/OCM-199

Signed-off-by: marcolan018 <llan@redhat.com>

### CODE REVIEW

**Comment by gdbranco** [comment]
SGTM

**Comment by marcolan018** [comment]
sample output  for `cluster describe`:
```
Private Hosted Zone:
 - ID:                      Z01166982JMJMXQ7PJQUV
 - Role ARN:                arn:aws:iam::962993289388:role/oadler-shared-vpc
```

File: cmd/describe/cluster/cmd.go [hunk 1]
```diff
@@ -335,6 +335,17 @@ func run(cmd *cobra.Command, argv []string) {
 			"OIDC Endpoint URL:          %s (%s)\n", str,
 			cluster.AWS().STS().OIDCEndpointURL(), managementType)
 	}
+	if cluster.AWS().PrivateHostedZoneID() != "" {
+		str = fmt.Sprintf("%s"+"Private Hosted Zone:\n", str)
+		str = fmt.Sprintf("%s"+
+			" - ID:                      %s\n", str,
+			cluster.AWS().PrivateHostedZoneID())
+		if cluster.AWS().PrivateHostedZoneRoleARN() != "" {
```
Comment (oriAdler) [inline cmd/describe/cluster/cmd.go:343]: This check is redundant, upon cluster creation we validate that both `privateHostedZoneID` and `privateHostedZoneRoleArn` are provided, so if `PrivateHostedZoneID() != ""` then `PrivateHostedZoneRoleARN()` must have a value.

Comment (oriAdler) [reply cmd/describe/cluster/cmd.go:343]: We can alternatively check this condition in the same `if` statement in line `338`.

**Review by oriAdler — COMMENTED** [review]
Left a small comment, other than that LGTM.

File: cmd/describe/cluster/cmd.go (hunk 1 above)
Comment (marcolan018) [reply cmd/describe/cluster/cmd.go]: @oriAdler 
I removed the redundant check in the change
Please help to review this PR again, thanks

**Pull request merged**